README.md

```diff
@@ -511,3 +511,30 @@ This option does not need to stop and restart the docker-containers:
 cd "$BTCPAY_BASE_DIRECTORY/btcpayserver-docker"
 ./backup.sh --only-db
 ```
+
+## How can I connect to the database?
+
+On the server you can open a database session by connecting via `psql` as the postgres user:
+
+```bash
+docker exec -ti $(docker ps -a -q -f "name=postgres_1") psql -U postgres
+```
+
+Then, inside `psql` you can select a database and interact with the tables:
+
+```bash
+# list databases
+\l
+
+# connect to database
+\c btcpayservermainnet
```

Comment:
and then a `postgres-btcpay.sh` that takes the network env var and connects to the db automatically

Reply:
imho a script would be rather short and overkill – as you can see by the instructions it isn't hthat much and I just wanted to document the steps, because they might be handy.